### DISCUSSION TITLE
메인 페이지 개선 아이디어 모집

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#626] [Ideas]
현재 학습로그 목록으로 구성되어있는 메인 페이지를 개선하려고 합니다!
메인 페이지 개선에 대한 의견을 자유롭게 남겨주세요~
회의는 다음주 주간회의(12월 20일)에 진행하겠습니다!

**Comment by gracefulBrown** [comment]
예시)
- 주간 읽음, 좋아요 학습로그 순위
- 주간 검색어 순위

**Comment by pkeugine** [comment]
* 주간 좋아요 가장 많이 얻은 크루 순위
* 주간 학습로그 많이 올린 크루 순위

**Comment by qhals321** [comment]
- 직군별(프론트, 백앤드) 핫한 태그가 떴음 좋겠어요~!  (예시: 스프링, 자바8, 리액티브)
- 미션도 하나의 태그로 가져가도 괜찮지않을까요!

**Comment by sunhpark42** [comment]
- 이모저모 명예의 전당 (글 많이 쓴 사람, 좋아요를 가장 많이 누른 사람 등등)

**Comment by sunhpark42** [comment]
https://www.figma.com/proto/15qD4XpMvnbBNDMJbzCjLn/Untitled?node-id=2%3A3&scaling=scale-down&page-id=0%3A1
대략 이것저것 해본 내용

**Reply by sunhpark42** [reply]
메인 학습로그 목록에 그림이 들어가면 좋을 것 같은데, 이미지가 꼭 들어가는 로그가 아니라서 조금 애매합니다.

**Reply by sunhpark42** [reply]
리포트는 메인페이지에서 뭘 보여줄지 생각해봐야겠어요

**Comment by sunhpark42** [comment]
배너 관련 참고
https://www.inflearn.com/
* 게시글
* 공지성 내용
* 밈

**Reply by sunhpark42** [reply]
학습로그 사용법

**Reply by sunhpark42** [reply]
4기 닉네임 롤링

**Reply by sunhpark42** [reply]
오늘의 생일자!

**Reply by sunhpark42** [reply]
초기 배너 내용 3가지 정하고 만든 후에, 컨텐츠 정하고 후에 재배포 하기로

**Reply by sunhpark42** [reply]
1. 프롤로그 소개
2. 프롤로그 이용법 (학습로그 작성법, 역량 이용법 등)
3. 4기 환영인사

**Comment by sunhpark42** [comment]
명예의 전당 -> 레벨별 시상식으로 만들어도 좋을듯. 정산느낌!

**Reply by sunhpark42** [reply]
FE/BE 구분 되는게 좋을 것 같음

**Comment by sunhpark42** [comment]
프론트 / 백엔드 구분은 어떻게?
- 멤버정보에 FE/BE 추가
- FE 점수에 *2

**Reply by sunhpark42** [reply]
일단 통합해두고, 한쪽으로 너무 치우치는 것 같으면 그때 구분하기

**Reply by sunhpark42** [reply]
태그, 학습로그, 리포트 등

**Reply by sunhpark42** [reply]
판단기준: 컨텐츠 목록, 사용자 로그 등.
처음은 가내수공업, 이후에 자동화 생각해보기

**Comment by sunhpark42** [comment]
리포트 기능 개발 미뤄질듯
- 주간 순위를 넣는것도 좋을 것 같다.

**Reply by sunhpark42** [reply]
1,2,3,4,5 도 있으면 좋을듯

**Comment by sunhpark42** [comment]
학습로그 썸네일
- 기본 제공 썸네일?!  자바, 스프링, 이런 큰 주제로 썸네일 주는 건 어때여??
- 작성자 프로필 있으면 좋을듯
- 본문 내용 덜 노출되게 수정
<img width="440" alt="스크린샷 2022-01-09 오후 8 00 03" src="https://user-images.githubusercontent.com/67677561/148679408-56926572-c0fe-45ce-a8d6-ad956d0ea24b.png">

**Reply by sunhpark42** [reply]
랜덤 썸네일 제공 -> 컬러, 태그 이용
